functions/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "main": "lib/index.js",
   "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
Is this ever used?

Reply:
Check `express.ts`. Its used as middleware. However, from what I've noticed, it fails to work in its deployed environment. If we were able to get `body-parser` to work it would obviate the need to do `JSON.parse()`.